src/server/auth/cmds/cmds_test.go

```diff
@@ -417,3 +417,41 @@ func TestRotateRootToken(t *testing.T) {
 		pachctl auth whoami | match "pach:root"
 	`).Run())
 }
+
+// TestPlurals walks through the command tree for each resource and verb combination defined in PPS.
+// A template is filled in that calls the help flag and the output is compared. It seems like 'match'
+// is unable to compare the outputs correctly, but we can use diff here which returns an exit code of 0
+// if there is no difference.
+func TestPlurals(t *testing.T) {
+	pluralCheckTemplate := `
+		pachctl auth {{VERB}} {{RESOURCE_PLURAL}} -h > plural.txt
+		pachctl auth {{VERB}} {{RESOURCE}} -h > singular.txt
+		diff plural.txt singular.txt
+		rm plural.txt singular.txt
+	`
+
+	resources := map[string][]string{
+		"repo": {"check", "set", "get"},
+	}
+
+	pluralMap := map[string]string{
+		"repo": "repos",
+	}
+
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c, _ := minikubetestenv.AcquireCluster(t)
```

Comment:
If we aren't making calls to a pachd instance, acquiring a cluster client shouldn't be necessary. If so, you can use `BashCmd` instead of `PachctlBashCmd` instead

Reply:
Ahh cool, will do!